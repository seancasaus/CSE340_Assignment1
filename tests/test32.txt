t1323 (((x)|((y).(z)))*).((p).(q)) , jdwrq3 (((x).((y).(z)))*).((((p).(q))*).(q)) , kj342 ((x)*).(((y)|(z)).((p)|(w))) #
" xxyzpqxyzpqqxzpxxxyw "