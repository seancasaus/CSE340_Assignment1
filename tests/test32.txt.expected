t1323 , "xxyzpq"
jdwrq3 , "xyzpqq"
kj342 , "xzp"
kj342 , "xxxyw"
